scio-bigquery/src/main/scala/com/spotify/scio/bigquery/types/TypeProvider.scala

```diff
@@ -220,6 +222,18 @@ private[types] object TypeProvider {
       q"override def toTableRow: ($name => ${p(c, GModel)}.TableRow) = ${p(c, SType)}.toTableRow[$name]")
   }
 
+  /** Enforce macro is not enclosed by package, therefore should be enclosed by class/object etc. */
```

Comment:
"Enforce that the macro is not enclosed by a package, but a class or object instead."